app/src/main/java/com/swent/assos/model/data/Event.kt

```diff
@@ -5,7 +5,7 @@ import android.media.Image
 data class Event(
     val title: String,
     val association: Association,
-    val image: Image,
+    val image: Image?,
```

Comment:
You shouldn't allow the variable image to be nullable. It it better to enter an empty string rather than a null image otherwise you will always have to check if the image is null or not.